### PR TITLE
Minor Change on Json File, frmmain, and Data base

### LINKED ISSUE #8124: Experiments Menu Name
Hello @N-thony, 

I notice that in the menu bar it is called the Experiments menu but in the View Menu it is called Options by Context Menu. I think these should be consistent, though I am not sure which is correct. Perhaps you know? Should they both be Experiments or both be Options by Context?

If you know please update accordingly, if not I can reach out and find out. 

Thanks, 

Rachel

### PR DESCRIPTION
**Pull request by MeSophie**
Fixes #8124
@lloyddewit This PR has some data base change. I change the text Option By Context Menu on View Menu by Experiments.
Please can you have a look on it ?

image

### CODE REVIEW

File: instat/frmMain.Designer.vb [hunk 1]
```diff
@@ -5107,9 +5107,7 @@ Partial Class frmMain
         Me.IsMdiContainer = True
         Me.MainMenuStrip = Me.mnuBar
         Me.Name = "frmMain"
-        Me.Text = "R-Instat  " + My.Application.Info.Version.Major.ToString + "." +
-                My.Application.Info.Version.Minor.ToString + "." +
-                My.Application.Info.Version.Build.ToString
+        Me.Text = "R-Instat  .."
```
Comment (lloyddewit) [inline instat/frmMain.Designer.vb:5110]: @MeSophie Was this change deliberate? I know that @ChrisMarsh82 fixed something related to this recently. You may want to check with him

**Review by lloyddewit — CHANGES_REQUESTED** [review]

File: instat/frmMain.Designer.vb (hunk 1 above)
Comment (MeSophie) [reply instat/frmMain.Designer.vb:5110]: @lloyddewit I also change the text in frmMain design because the name of the `Experiments` menu on View was still `Options By Context Menu`.

Comment (lloyddewit) [reply instat/frmMain.Designer.vb:5110]: yes, I'm happy with that change.

My question is about line 5110. If you look at PR #8100 , you'll see the change below. This PR reverses that (which I believe is incorrect). I suspect that this PR is branched from an older version of main. Please could you fix this?
(I hope this makes sense, if not please ask @N-thony or myself)
thanks

image

Comment (MeSophie) [reply instat/frmMain.Designer.vb:5110]: @lloyddewit  I checked this with Antoine I you are right. Please it is possible to you to remove this file from the PR. If not I will just close this PR and redo the changes.

Comment (lloyddewit) [reply instat/frmMain.Designer.vb:5110]: Instructions to 'remove' a file from a PR are [here](https://github.com/africanmathsinitiative/R-Instat/wiki/GitHub-notes).
Or if you prefer, you could make a new PR.
In either case, you may wish to wait until PR #8130 from @anastasia-mbithe is merged. Otherwise you will get a merge conflict.

Comment (MeSophie) [reply instat/frmMain.Designer.vb]: @lloyddewit I think I resolve the problem. I copy  the code and paste in VB.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@berylwaswa if you can test, then we can merge, thanks

**Comment by berylwaswa** [comment]
@lloyddewit @MeSophie This is fine

**Review by berylwaswa — APPROVED** [review]
